tests/chocolatey-tests/commands/choco-search.Tests.ps1

```diff
@@ -35,7 +36,8 @@ Describe "choco <_>" -ForEach $Command -Tag Chocolatey, SearchCommand, FindComma
         Remove-ChocolateyTestInstall
     }
 
-    Context "Searching packages with no filter (Happy Path)" {
+    # Skip on Licensed Extension when searching against a v3 source as our source is not returning consistent results.

```

Comment:
I know we already spoke about this, but these comments need updated as we're doing this in CLI and not just CLE :)

Reply:
I've updated these comments and rebased everything onto the head of the release branch 😄